automation/balena-push-os-version.sh

```diff
@@ -10,7 +10,7 @@ echo "[INFO] Starting docker."
 dockerd --data-root /scratch/docker > /var/log/docker.log &
```

Comment:
Let's reword the commit log saying `push none hostapp type of images` to something like `push non-hostapp type of images` to make it clearer

Reply:
This was already done when I rebased, so resolving.